src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -22,6 +22,17 @@
             <tag name="cache.pool" />
         </service>
 
+        <service id="cache.adapter.system" class="Symfony\Component\Cache\Adapter\ChainAdapter" abstract="true">
```

Comment:
What about using the interface as the class here as the service can also just be a `FilesystemAdapter`?